FileUploader/FileUpload.cs

```diff
@@ -1,83 +0,0 @@
-﻿/*
- * Minio .NET Library for Amazon S3 Compatible Cloud Storage, (C) 2017 Minio, Inc.
```

Comment:
The FileUploader project is on the SDK to give a quick running example to new/inexperienced users to demo basic functionality - imo, we should keep this example.

Reply:
@poornas I can add it back, I didn't think it was adding any value and was just noise. We already  have 3 test projects and this should be combined down to one. This should go in a test or snippet someplace else. There are already sample code in the test project that contains every operation.

Reply:
Thoughts on this?